modules/core/05-port/types/ibc_legacy_module.go

```diff
@@ -22,13 +22,13 @@ func (im *LegacyIBCModule) GetCallbacks() []ClassicIBCModule {
 
 // NewLegacyIBCModule creates a new IBCModule given the keeper
 func NewLegacyIBCModule(cbs ...ClassicIBCModule) ClassicIBCModule {
-	return LegacyIBCModule{
+	return &LegacyIBCModule{
```

Comment:
small tweak since we need the reference type to implement the interface, not the value type.